### PR TITLE
Hungarian Translation

### PR DESCRIPTION
**Pull request by TBarnus30**
I am a Hungarian high school student who made a translation from English-->Hungarian.
---
man this took a while.
Translating the heads in html, then the .md, checking, fixing some stuff, and me being stupid and not knowing how to make a pull request.
There may still be a few typos or mistakes and I am sorry in advance if that happens. I tried my best.
---
Thank you. Thank you Nicky Case, for making awesome games like Adventures with Anxiety and incredible readables like this one. I show my gratitude to you by trying to make a fan-made translation (an amateur one, but still).
---
I sometimes have thoughts like "I think I may have anxiety, but I'm not 100% sure, and what if I'm just overdramatizing everything? I don't have a say in anything that happens in the AwA community because I myself may not have anxiety."
Maybe I overthink things. Maybe I'm emotional.
But I think I might have grown closer to my own wolf while making this translation, and showing that it *does* care about me.
It just wants to protect me.
---
Thank you and sorry for having to read this letter. This is kind of like a vent-letter (if that makes sense?), but i just had to write these down somewhere, even if it's directed to a person I barely even know.
---
I wish you all the best, be it now, or in the future.

Best wishes,
TBarnus30 (and his wolf, probably).
🐺

### CODE REVIEW

**Comment by balazsjonas** [comment]
Don't worry @TBarnus30 , it's a great job
I will check your translation.
Don't you want to squash the the changes into a single commit?


By the way I would add the Hungarian crisis hotlines in addition to or instead of the original. Do you agree, @ncase ?
Eg.[https://egeszsegvonal.gov.hu/maradj-egeszseges/lelkisegely-szolgalatok.html](https://egeszsegvonal.gov.hu/maradj-egeszseges/lelkisegely-szolgalatok.html)

**Comment by TBarnus30** [comment]
Thank you for the comment!
Yes, if we could squash it into a single one it would be great.
I'm not really a pro at using Github.
Also yes, we could add Hungarian crisis hotlines to the ones already listed. These are all great additions.

**Comment by TBarnus30** [comment]
Okay,
- Added Hungarian crisis hotline page and number,
- Squished commits into one

File: hu.html [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+        <strong>
+          <a href="#toc_2">A Fájdalom Értelme, az Érzelmek Funkciója</a>
+        </strong> (Teória)</li>
+      <li>
+        <strong>
+          <a href="#toc_3">A Szokások Alapjai</a>
+        </strong> (Teória + praktikus dolgok)</li>
+      <li>
+        <strong>
+          <a href="#toc_4">9 Bizonyítékkal alátámasztott Mentális Egészségügyi Szokás</a>
```
Comment (balazsjonas) [inline hu.html:70]: Egyszerüen csak "Bizonyítékkal alátámasztott jó szokások", még gondolkodom rajta.

File: hu.html [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+    <p>
+      <em>Rajongók által készített fordítások: <a href="de.html">Német</a>・<a href="https://github.com/ncase/mental-health#how-to-translate">(Csinálj sajátot?)</a></em>
+    </p>
+    <hr />
+    <h1 id="toc_0">Egy régi barát</h1>
+    <p>Emlékszem az első pánikrohamomra. A buszmegállónál álltam. Most költöztem egy új városba, és nem jött meg a busz. Szóval az agyamban lévő kis farkas azt mondta nekem:</p>
+    <blockquote>
+      <p>🐺 Egy új helyen vagyunk, és még csak azt se tudjuk kitalálni hogy működnek itt a buszok?</p>
+      <p>🐺 Ez azt jelenti hogy nem fogunk tudni barátokat szerezni! Vagy hazaérni épségben! Vagy a túléléshez szükséges dolgokat megvenni!</p>
+      <p>🐺 Várjunk, miért dobog a szívünk? Miért nem tudunk lélegezni?! Nem úgy van hogy a családban fut a szívroham??!?! Jaj nekem, ez lenne a vég?!?!?!</p>
```
Comment (balazsjonas) [inline hu.html:94]: dobog --> kalapál, dübörög vagy valami súlyosabb.

File: hu.html [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+      <p>🐺 ÁÁÁÁÁÁÁÁÁÁÁÁÁÁ</p>
+    </blockquote>
+    <p>4 mentális lépés alatt, egy nem megérkező busz miatt 100%-ig <em>azt hittem</em> hogy meg fogok halni.</p>
```
Comment (balazsjonas) [inline hu.html:97]: nem megérkező -> meg nem érkező
esetleg 100%ig helyett "100% biztos voltam benne, hogy"

File: hu.html [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+    <p>Szóval, ezért nem bíztam soha a mondásban, hogy "bízz a megérzéseidben." Vagy, hogy "az érzelmeid érvényesek." Képzeld el ezt a szituációt:</p>
```
Comment (balazsjonas) [inline hu.html:98]: Esetleg "érzelmeid megalapozottak"?

File: hu.html [hunk 5]
```diff
@@ -0,0 +1,438 @@
...
+    <blockquote>
+      <p>Barát, mondja neked: Értéktelennek érzem magamat. Úgy érzem hogy csak egy teher vagyok mindenki számára. Úgy érzem, hogy mélyen, én egy visszafordíthatatlanul törött, gonosz ember vagyok.</p>
```
Comment (balazsjonas) [inline hu.html:100]: törött -> megtört, összetört

File: hu.html [hunk 6]
```diff
@@ -0,0 +1,438 @@
...
+    </blockquote>
+    <blockquote>
+      <p>Te, mondod a barátodnak: Az érzelmeid érvényesek! :D</p>
+    </blockquote>
+    <p>Ezért, én elhittem azokat a <em>más</em> mondásokat: "Az érzelmek irracionálisak." "Az egyetlen dolog amitől félni kell az maga a félelem." "Ne aggódj, légy boldog!"</p>
```
Comment (balazsjonas) [inline hu.html:105]: mondások -> közghelyeket?

**Review by balazsjonas — CHANGES_REQUESTED** [review]
I will continue from here, I have to go to work. Most comments are only suggestions, not mandatory. Just think about it.

File: hu.html [hunk 7]
```diff
@@ -0,0 +1,438 @@
...
+    <p>Szerencsére, a barátaim féltek <em>helyettem</em>, amikor túlságosan féltem hogy megengedjem magamnak hogy féljek. (Csak a félelemtől kell félni, igaz?) A baráraim segítettek rájönni arra, hogy a szituációm <em>igenis</em> bántalmazó volt, és kisegítettek belőle.</p>
+    <p>Egy pár hónappal később, egy régi barát ismerős hangját hallottam:</p>
+    <blockquote>
+      <p>🐺 ÁÁÁÁÁÁÁÁÁÁÁÁÁÁ</p>
+    </blockquote>
+    <hr />
+    <h1 id="toc_1">Kipróbált Terápiák</h1>
+    <p>Ha ez egy film lenne, az lett volna a pont amikor rájöttem a félelem értékére.</p>
+    <p>Nem. Nem tanultam semmit.</p>
+    <p>Viszont, ez arra inspirált hogy <em>végre</em> kipróbáljam a pszichoterápiát. Előszőr a <a href="https://en.wikipedia.org/wiki/Cognitive_behavioral_therapy">Kognitív viselkedési terápiát</a> próbáltam ki (CBT), aminek <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/da.20829">metaanalízisei kimutatták</a> hogy ugyanolyan jó vagy akár jobb mint a gyógyszerek depresszió &amp; aggodalom ellen. A terápia azt ajánlja, hogy "hívd ki a gondolataidat" azzal hogy rámutatsz a sok irracionális "kognitív torzulásra" a félelmeidben. (🤓 “Hát, <em>Igazából</em>...”)</p>
```
Comment (balazsjonas) [inline hu.html:118]: Itt határozotta "szorongás" az "aggodalom" helyett.

File: hu.html [hunk 8]
```diff
@@ -0,0 +1,438 @@
...
+    <p>A CBT sokat segített rajtam, de gyorsan problémába ütköztem. A problémám az volt, hogy a CBT-t <em>fegyverként</em> használtam a félelmem ellen. Még mindig úgy gondoltam a félelemre mintha az <em>ellenségem</em> lenne.</p>
+    <p>Utána, kipróbáltam egy <a href="https://en.wikipedia.org/wiki/Mindfulness">éberség</a>-alapú terápiát – pontosabban, <a href="https://en.wikipedia.org/wiki/Acceptance_and_commitment_therapy">Elfogadás &amp; Elköteleződés terápia</a> – ami azt ajánlja hogy <em>ne</em> harcolj a gondolataiddal, hanem csak engedd hogy az érzelmek elhaladjanak melletted. (Metafóra amit mondtak nekem: az aggodalom olyan mint a futóhomok – minél többet küzködsz, annál gyorsabban süllyedsz. Ezért: ne küzködj.)</p>
```
Comment (balazsjonas) [inline hu.html:120]: Itt használhatnánk a magyar linket, ha már feltehetően magyarok olvassák, de meghagynám a címkének a mindfulness-t.  Úgy ismertebb

https://hu.wikipedia.org/wiki/Tudatos_jelenl%C3%A9t

és a párja csak egy bekezdés:
https://hu.wikipedia.org/wiki/Tudatos_jelenl%C3%A9t#Elfogad%C3%A1s_%C3%A9s_elk%C3%B6telez%C5%91d%C3%A9s_ter%C3%A1pia

File: hu.html [hunk 9]
```diff
@@ -0,0 +1,438 @@
...
+    <p>Megintcsak, úgy használtam az éberséget mintha csak egy másik mód lenne arra, hogy <em>ne figyeljek oda</em> a félelmemre. A félelem még <em>mindig</em> az ellenségem volt.</p>
```
Comment (balazsjonas) [inline hu.html:121]: inkább "tudatos jelenlét", mint "éberség", ha mindenképp le szeretnénk fordítani a mindfulness-t,

**Review by balazsjonas — CHANGES_REQUESTED** [review]
to be continued

**Comment by TBarnus30** [comment]
Köszi a javaslatokat! Szerintem mindegyik nagyszerű, hozzá is adtam azokat a fájlokhoz. (Kivéve a "Bizonyítékkal alátámasztott jó szokások"-at, mert mondtad hogy azon még gondolkodsz.)

**Comment by ncase** [comment]
Hi @TBarnus30 ! Thank you for your kind words, and putting in all the effort to making this available to more folks. (and thank you @balazsjonas for polishing it up!)

And, uh, sincerest apologies to you both that it took me so long to finally get around to merging the pull request. 😬 My email is a nightmare, a torture device of my own design.

**Pull request merged**